src/fn/smf.ts

```diff
@@ -0,0 +1,99 @@
+import type { AnySoupElement, PcbSilkscreenPath } from "circuit-json"
+import { z } from "zod"
+import { rectpad } from "../helpers/rectpad"
+import { silkscreenRef, type SilkscreenRef } from "src/helpers/silkscreenRef"
+import { length } from "circuit-json"
+
+export const smf_def = z.object({
+  fn: z.string(),
+  num_pins: z.literal(2).default(2),
+  w: z.string().default("4.80mm"),
```

Comment:
Can you share in the kicad-viewer from where are you measuring so that the width is 4.8?

Reply:
I don't think 4.80mm is matching. Can you look more into this? I think there is a potential bug somewhere or miss calculation